src/components/pages/order/main/Admin/AdminPanel/AddProductForm.js

```diff
@@ -0,0 +1,88 @@
+import { useContext, useState } from "react"
+import styled from "styled-components"
+import OrderContext from "../../../../../../context/OrderContext"
+import TextInput from "../../../../../reusable-ui/TextInput"
+import Button from "../../../../../reusable-ui/Button"
+import ImagePreview from "./ImagePreview"
+import SubmitMessage from "./SubmitMessage"
+import { getInputTextsConfig } from "./inputTextConfig"
+
+export const EMPTY_PRODUCT = {
+  id: "",
+  title: "",
+  imageSource: "",
+  price: 0,
+}
+
+export default function AddProductForm() {
+  const { handleAddProduct, newProduct, setNewProduct } = useContext(OrderContext)
+  const [isSubmitted, setIsSubmitted] = useState(false)
+
+  const handleChange = (event) => {
+    setNewProduct({
+      ...newProduct,
+      [event.target.name]: event.target.value,
+    })
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    handleAddProduct({
+      ...newProduct,
+      id: crypto.randomUUID(),
+    })
+    setIsSubmitted(true)
+    setTimeout(() => {
+      setIsSubmitted(false)
+    }, 2000)
+    setNewProduct(EMPTY_PRODUCT)
+  }
+
+const inputTexts = getInputTextsConfig(newProduct);
+
+  return (
+    <AddProductFormStyled onSubmit={handleSubmit}>
+      <ImagePreview imageSource={newProduct.imageSource} title={newProduct.title}/>
+      <div className="input-fields">
+        {inputTexts.map((input) => 
```

Comment:
```suggestion
        {inputTexts.map((inputText) => 
```

optionnel (c'est seulement si on est vraiment rigoureux).